schemas/Manual.locations.schema.json

```diff
@@ -46,56 +42,59 @@
                     },
                     "uniqueItems": true
                 },
-                "place_item_category": {
-                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
+                "region": {
+                    "description": "(Optional) The name of the Region this location is part of.",
+                    "type": "string"
+                },
+                "place_item": {
+                    "description": "(Optional) Specify the list of item that can be placed at this location (will only randomly choose 1).",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item_category"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
-                "place_item": {
-                    "description": "(Optional) Specify the list of item that can be placed at this location (will only randomly choose 1).",
+                "dont_place_item": {
+                    "description": "(Optional) Specify the list of items that cannot be placed at this location.",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
-                "dont_place_item_category": {
-                    "description": "(Optional) Specify the categories of items that will never be placed at this location.",
+                "place_item_category": {
+                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
```

Comment:
Does this ignore logic? Eventually the multiworld will still check if the game is beatable.

Also we should probably include "if multiple categories are given, the chosen item is from ANY category, not ALL categories"

Reply:
Yeah, this is more warning the world builder that their place item will succeed even if it would cause an unreachable location during generation. i.e., it doesn't check to see that the place item itself would prevent reaching the location. Seems like a good warning to have, so preserved that with a minor adjustment.

Also, agree, tweaked the wording on items and item categories to more clearly indicate that 1 item is being picked from the matching criteria.